components/permissions/permission_widevine_utils.cc

```diff
@@ -0,0 +1,31 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/permissions/permission_widevine_utils.h"
+
+#include "brave/components/constants/pref_names.h"
+#include "components/permissions/permission_request.h"
+#include "components/permissions/request_type.h"
+#include "components/prefs/pref_service.h"
+
+namespace permissions {
+
+void AskWidevineInstall(PrefService* prefs, bool ask) {
```

Comment:
As discussed in dm the others naming should be consistent. It's confusing and hard to read when we use different terminology to refer to the same thing 
`SetAskEnableWidevine`

Reply:
I removed the whole method.